client/src/types/guild.ts

```diff
@@ -0,0 +1,44 @@
+import { ContractAddress, ID } from "@bibliothecadao/eternum";
+
+export type GuildAndName = {
```

Comment:
why do you need so many different types for guild ? would be more clear if you could have one GuildInfo type. Also move it to sdk